activesupport/lib/active_support/core_ext/yaml.rb

```diff
@@ -0,0 +1,17 @@
+require "yaml"
+
+module YAML
```

Comment:
I sure can but fear that we might want to expand this to more `yml` files over time. Would it be OK if I make it a module of its own rather than a monkey patch? `ActiveSupport::YamlWithSharedSection.load`?